wagtail/admin/views/pages/copy.py

```diff
@@ -22,8 +22,8 @@ def copy(request, page_id):
     # Check if the user has permission to publish subpages on the parent
     can_publish = parent_page.permissions_for_user(request.user).can_publish_subpage()
 
-    # Create the form
-    form = CopyForm(
+    form_class = getattr(page.specific_class, "copy_form_class", CopyForm)
```

Comment:
Just a reminder that there will need to be a test for this override added. 

Maybe even with the code of a custom form that you have in the docs.

Reply:
I've added a test. Can you please review it?